Cargo.toml

```diff
@@ -35,6 +35,7 @@ all = { level = "warn", priority = -1 }
 missing-const-for-fn = "allow" # TODO: https://github.com/rust-lang/rust-clippy/issues/14020
 use-self = "warn"
 redundant-clone = "warn"
+result_large_err = "allow"
```

Comment:
which functions are currently marked as large?

if those are in a hot path, worth checking if we should box the err instead

Reply:
It's this error
https://github.com/alloy-rs/trie/blob/main/src/proof/error.rs#L7

I don't think we use this in reth anywhere